src/test/java/com/google/javascript/gents/multiTests/module_imports/export_default.ts

```diff
@@ -1,3 +1,3 @@
-export default class A {
```

Comment:
It wasn't immediately obvious how to solve/fix.  

The jscomp code generator seems to decide whether to keep it or remove it based on if it's used in the file (there should be a test in here to verify this behavior)

Is this something that you would like to see fixed in this PR or as a future feature request?